demo/index.html

```diff
@@ -41,7 +45,7 @@
 		<h3>cosmoz-bottom-bar-view with less content</h3>
 		<demo-snippet>
 			<template>
-				<cosmoz-viewinfo class="fit layout vertical">
+				<cosmoz-viewinfo class="fit layout vertical reduce-height">
```

Comment:
@plequang Can you have a look to this PR let me know if anything you want to change?